cmd/notation/sign.go

```diff
@@ -52,33 +53,24 @@ func signCommand(opts *signOpts) *cobra.Command {
 		Long: `Sign artifacts
 
 Note: a signing key must be specified. This can be done temporarily by specifying a key ID, or a new key can be configured using the command "notation key add"
-
+		
 Example - Sign an OCI artifact using the default signing key, with the default JWS envelope, and use OCI image manifest to store the signature:
-  notation sign <registry>/<repository>@<digest>
+	notation sign <registry>/<repository>@<digest>
 
 Example - Sign an OCI artifact using the default signing key, with the COSE envelope:
-  notation sign --signature-format cose <registry>/<repository>@<digest> 
+	notation sign --signature-format cose <registry>/<repository>@<digest> 
 
 Example - Sign an OCI artifact with a specified plugin and signing key stored in KMS 
-  notation sign --plugin <plugin_name> --id <remote_key_id> <registry>/<repository>@<digest>
+	notation sign --plugin <plugin_name> --id <remote_key_id> <registry>/<repository>@<digest>
 
 Example - Sign an OCI artifact using a specified key
-  notation sign --key <key_name> <registry>/<repository>@<digest>
+	notation sign --key <key_name> <registry>/<repository>@<digest>
 
 Example - Sign an OCI artifact identified by a tag (Notation will resolve tag to digest)
-  notation sign <registry>/<repository>:<tag>
+	notation sign <registry>/<repository>:<tag>
 
 Example - Sign an OCI artifact stored in a registry and specify the signature expiry duration, for example 24 hours
-  notation sign --expiry 24h <registry>/<repository>@<digest>
-
-Example - [Experimental] Sign an OCI artifact referenced in an OCI layout
-  notation sign --oci-layout "<oci_layout_path>@<digest>"
-
-Example - [Experimental] Sign an OCI artifact identified by a tag and referenced in an OCI layout
-  notation sign --oci-layout "<oci_layout_path>:<tag>"
-
-Example - [Experimental] Sign an OCI artifact and use OCI artifact manifest to store the signature:
-  notation sign --signature-manifest artifact <registry>/<repository>@<digest>
+	notation sign --expiry 24h <registry>/<repository>@<digest>
```

Comment:
Are the indentations tabs or spaces? Please make sure that they are consistent across all commands.